templates/https/agencies.html

```diff
@@ -1,33 +1,37 @@
 {% extends "layout.html" %}
 
 {% block page_id %}https{% endblock %}
-{% block title %}HTTPS in the federal government - By agency{% endblock %}
+{% block title %}Secure HTTP (HTTPS) - Pulse{% endblock %}
```

Comment:
Sorry to catch this late, but why did the <title> get renamed here? `HTTPS` is much preferred to `Secure HTTP (HTTPS)`. No one uses the phrase "Secure HTTP".